pom.xml

```diff
@@ -1,32 +1,39 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
-    <modelVersion>4.0.0</modelVersion>
-
-    <artifactId>kafka-metrics</artifactId>
-    <name>Kafka metric set</name>
-    <groupId>com.turn.kafka.metrics</groupId>
-    <version>1.0.2</version>
-
-    <packaging>jar</packaging>
-    <description>
+	<modelVersion>4.0.0</modelVersion>
+	<artifactId>kafka-metrics</artifactId>
+	<name>Kafka metric set</name>
+	<groupId>com.turn.kafka.metrics</groupId>
+	<version>1.0.3</version>
+	<packaging>jar</packaging>
+	<description>
         A set of classes which exposes Kafka's metrics to allow you to tailor them to your reporting mechanism.
     </description>
-
-    <properties>
-        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-        <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
-    </properties>
-
-    <dependencies>
-        <dependency>
-            <groupId>com.yammer.metrics</groupId>
-            <artifactId>metrics-core</artifactId>
-            <version>2.2.0</version>
-        </dependency>
-        <dependency>
-            <groupId>org.apache.kafka</groupId>
-            <artifactId>kafka-clients</artifactId>
-            <version>0.8.2.0</version>
-        </dependency>
-    </dependencies>
+	<repositories>
+		<repository>
+			<id>turn</id>
+			<url>http://maven.turn.com/content/repositories/turn</url>
+		</repository>
+	</repositories>
+	<parent>
+		<groupId>com.turn</groupId>
+		<artifactId>parent</artifactId>
+		<version>1.0.0</version>
+	</parent>
+	<properties>
+		<project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+		<project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
+	</properties>
+	<dependencies>
+		<dependency>
+			<groupId>com.yammer.metrics</groupId>
+			<artifactId>metrics-core</artifactId>
+			<version>2.2.0</version>
+		</dependency>
+		<dependency>
+			<groupId>org.apache.kafka</groupId>
+			<artifactId>kafka-clients</artifactId>
+			<version>0.8.2.0</version>
```

Comment:
If you are talking about kafka-clients version, not sure why it need to match, but if you talk about kafka-metrics version, I was instructed by Hemant to bump up the version, since he is going to incorporate the new version in.